README.md

```diff
@@ -1,28 +1,65 @@
-# dd-trace-js
+# dd-trace-js: Node.js APM Tracer Library
 
-[![npm](https://img.shields.io/npm/v/dd-trace.svg?colorB=blue)](https://www.npmjs.com/package/dd-trace)
-[![npm (tag)](https://img.shields.io/npm/v/dd-trace/dev.svg)](https://www.npmjs.com/package/dd-trace/v/dev)
+[![npm v3](https://img.shields.io/npm/v/dd-trace/latest?color=blue&label=dd-trace%40v3&logo=npm)](https://www.npmjs.com/package/dd-trace)
+[![npm v2](https://img.shields.io/npm/v/dd-trace/latest-node12?color=blue&label=dd-trace%40v2&logo=npm)](https://www.npmjs.com/package/dd-trace/v/latest-node12)
+[![npm dev](https://img.shields.io/npm/v/dd-trace/dev?color=orange&label=dd-trace%40dev&logo=npm)](https://www.npmjs.com/package/dd-trace/v/dev)
 [![codecov](https://codecov.io/gh/DataDog/dd-trace-js/branch/master/graph/badge.svg)](https://codecov.io/gh/DataDog/dd-trace-js)
 
-**Node.js APM Tracer**
 
-Datadog APM tracing client for Node.js.
+## Documentation
 
-## Getting Started
+Most of the documentation for `dd-trace-js` is available on these webpages:
 
-For a basic product overview, check out our [setup documentation](https://docs.datadoghq.com/tracing/languages/nodejs/).
+- [Tracing Node.js Applications](https://docs.datadoghq.com/tracing/languages/nodejs/) - most of the project documentation is over here
```

Comment:
Probably worth still explicitly calling out to go here for setup docs

Reply:
added